systemd/schema-registry.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Schema Registry
+Documentation=http://docs.confluent.io/current/schema-registry/docs/index.html
+Requires=network.target
+Wants=kafka-server.service
```

Comment:
Ok, that makes sense. Follow up if we're adding this proactively -- will it actually be called `kafka-server.service`, or just `kafka.service`?